src/test/java/org/kiwiproject/elk/ElkAppenderFactoryTest.java

```diff
@@ -34,6 +35,13 @@ void setUp() {
         appenderFactory = new AsyncLoggingEventAppenderFactory();
     }
 
+    @AfterEach
+    void tearDown() {
+        System.clearProperty("kiwi.elk.host");
```

Comment:
Here's a good place to try out JUnit Pioneer since it has annotations to set and clear system properties 👍 

I am not overly concerned, however, about adding it just for that in this case.

Reply:
Want me to add a TODO?

Reply:
No. Either add pioneer or don't. There is no TODO.